roles/devscripts/tasks/cleanup.yml

```diff
@@ -29,6 +29,23 @@
     path: "{{ cifmw_devscripts_repo_dir }}"
   register: devscript_repodir
 
+- name: Check logs for token expiration
+  when:
+    - devscript_repodir.stat.exists
+  ansible.builtin.command: >-
+    egrep -r "The token provided is invalid or expired" {{ cifmw_devscripts_logs_dir }}
```

Comment:
The logs of `dev-scripts` wouldn't be available in this location `cifmw_devscripts_logs_dir`. It would be available in `(cifmw_devscripts_repo_dir, 'logs') | path_join)` . In the `01_install_requirements...log`

Reply:
It would be more "Ansibley" to use the find module[1] with the `contains` parameter. We don't have any example in the case sadly but if it works I think that would be cleaner. 

[1] https://docs.ansible.com/ansible/2.7/modules/find_module.html

Reply:
Thanks for both comments, I've changed the proposed code to just check if directory exists without greping.